custom_components/kumo/translations/en.json

```diff
@@ -15,15 +15,34 @@
           "password": "[KumoCloud Password]",
           "prefer_cache": "[Prefer Local Cache]"
         }
+      },
+      "confirm": {
+        "title": "IP Assignment",
+        "description": "Kumo failed to return an IP for the following units, please replace the text with the local IP"
```

Comment:
Please separate this into 2 sentences, not spliced together with a comma. Also, I'd suggest "failed to return an IP address" and "with their local IP addresses"